src/providers/dart_commands.ts

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+import * as as from "../analysis/analysis_server_types";
+import * as editors from "../editors";
+import * as vs from "vscode";
+import { Analyzer } from "../analysis/analyzer";
+
+export class DartCommands  {
+	private analyzer: Analyzer;
+	constructor(context: vs.ExtensionContext, analyzer: Analyzer) {
+		this.analyzer = analyzer;
+
+        context.subscriptions.push(
```

Comment:
Not sure if it's GitHub (I'm not at PC to check), but the indenting in this file looks a bit wonky.